webapp/channels/src/components/purchase_modal/renewal_card.scss

```diff
@@ -0,0 +1,258 @@
+.PurchaseModal {
+    .RenewalRHS {
+        position: sticky;
+        display: flex;
+        width: 25%;
+        flex-direction: column;
+        padding-top: 80px;
+
+        .RenewalCard {
+            max-width: 353px;
+            padding: 28px 32px;
+            border: 1px solid var(--light-8-center-channel-text, rgba(61, 60, 64, 0.08));
+            background: var(--light-center-channel-bg, #fff);
+            border-radius: var(--spacing-xxxxs, 4px);
+            box-shadow: 0 2px 3px 0 rgba(0, 0, 0, 0.08);
+
+            .SeatsCalculator {
+                padding: 0;
+
+                .SeatsCalculator__seats-item.SeatsCalculator__seats-item--input {
+                    padding: 0;
+                }
+            }
+
+            .RenewalSummary {
+                width: 332px;
+                padding: 28px 32px;
+                border: 1px solid rgba(var(--sys-center-channel-color-rgb), 0.08);
+                margin-left: 20px;
+                background-color: var(--sys-center-channel-bg);
+                border-radius: 4px;
+                box-shadow: 0 2px 3px rgba(0, 0, 0, 0.08);
+            }
+
+            .RenewalSummary__noBillingHistory {
```

Comment:
nit: using BEM for css you can always use `&__noBillingHistory` and the same for all the children of `.RenewalSummary`